atd-vzv/src/views/map/Map.js

```diff
@@ -257,30 +233,20 @@ const Map = () => {
       {buildHighInjuryLayer(overlay)}
       {!!cityCouncilOverlay && overlay.name === "cityCouncil" && (
         <Source type="geojson" data={cityCouncilOverlay}>
-          {/* Add beforeId to render beneath crash points */}
-          <Layer beforeId="base-layer" {...cityCouncilDataLayer} />
+          {/* Add beforeId to render beneath crash points, road layer, and map labels */}
+          <Layer beforeId="road-street" {...cityCouncilDataLayer} />
```

Comment:
really cool that you were able to get this to overlay placed nicely underneath the street layer. Its a great improvement visually.

Reply:
Thanks - I found all the layers and their IDs hidden away in the map instance. 👍